packages/web-components/src/components/va-checkbox/va-checkbox.tsx

```diff
@@ -239,7 +239,7 @@ export class VaCheckbox {
               disabled={disabled}
               onChange={this.handleChange}
             />
-            <label htmlFor="checkbox-element" id="option-label" class="usa-checkbox__label">
+            <label htmlFor="checkbox-element" id="option-label" class="usa-checkbox__label" part="label">
               {label}&nbsp;
               {required && <span class="usa-label--required">{i18next.t('required')}</span>}
               {checkboxDescription && <span class="usa-checkbox__label-description" aria-describedby="option-label">{checkboxDescription}</span>}
```

Comment:
I think that you will also want to add `part` to the description element too. Otherwise all of the content wrapped by the label will be bold like this:

![Screenshot 2024-03-21 at 12 19 29 PM](https://github.com/department-of-veterans-affairs/component-library/assets/872479/cfe7e28f-25df-4c7a-9694-53788edb71f0)

Once that's done, then there will be more flexibility with the weights:

```css
va-checkbox::part(label) {
  font-weight: bold;
}

va-checkbox::part(description) {
  font-weight: normal;
}
```

![Screenshot 2024-03-21 at 12 21 30 PM](https://github.com/department-of-veterans-affairs/component-library/assets/872479/aab76123-a9b6-41a3-a01a-3cbdcd3b977d)

Reply:
@jamigibbs just updated!